nailgun/entities.py

```diff
@@ -1096,6 +1135,23 @@ def __init__(self, server_config=None, **kwargs):
         }
         super(ContentView, self).__init__(server_config, **kwargs)
 
+    def read(self, entity=None, attrs=None, ignore=()):
+        """Fetch an attribute missing from the server's response.
+
+        For more information, see `Bugzilla #1237257
+        <https://bugzilla.redhat.com/show_bug.cgi?id=1237257>`_.
+
+        """
+        if (getattr(self._server_config, 'version', parse_version('6.1')) <
```

Comment:
Yes, the API changes with each new release of Satellite. Sometimes bugs are fixed with a new release, so we conditionally execute a workaround only for an older release. Sometimes an entity's fields change, so we conditionally add those new fields only for a new version. And so on.